Microsoft.Toolkit.Uwp.PlatformSpecificAnalyzer/Microsoft.Toolkit.Uwp.PlatformSpecificAnalyzer.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework><Title>Windows Community Toolkit UI</Title>
+    <Description>This .NET standard library provides analyzer and code fixer to ensure that version / platform specific code is well guarded. It is part of the Windows Community Toolkit.</Description>
+    <PackageTags>UWP Toolkit Windows</PackageTags>
+    <Title>Windows Community Toolkit UWP Platform Specific Analyzer</Title>
+
+    <!-- This is a temporary workaround for https://github.com/dotnet/sdk/issues/955 -->
+    <DebugType>Full</DebugType>
```

Comment:
Haven't this been fixed in the latest VS? Oren even opened an issue to remove the workaround on other Toolkit projects https://github.com/Microsoft/WindowsCommunityToolkit/issues/1951

Reply:
I'm not sure. @nmetulev asked me to make it same as other projects so I just copied that bit as was

Reply:
@onovotny, have you verified this is fixed now - should we remove it?

Reply:
This could be merged today and a separate PR cleaning up all projects would be a good idea if applicable

Reply:
I don't see a problem. Let's merge it.